javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,120 @@
+/* My age in 2027*/ 
+let yearOfBerth;
```

Comment:
try to keep the spelling correct -> yearOfBirth :)
It'll be useful when you work in a team, and your fellow devs look search the code